AcademyApp/ViewControllers/LoginViewController.swift

```diff
@@ -10,68 +10,73 @@ import UIKit
 
 // MARK: - View Controller for login scree
 
-class LoginViewController: UIViewController
-{
+class LoginViewController: UIViewController {
+
+    // MARK: - Stored Properities
+
+    // default vertical spacing constraint constant between doneButton and login form
+    private var defaultVerticalSpacingConstant: CGFloat = 0
```

Comment:
Now you have one constant in enum config and second alone as property... Would be nice to have both in config and I would advice to rename config to UIConfig for example bc config is common name for anything :-D

Reply:
Renamed Config to UIConfig. Move all of the constants to UIConfig enum.